src/Object3D.cpp

```diff
@@ -30,9 +28,8 @@ void Object3D::insertNewObject(std::shared_ptr<Object3DObs> new_object) {
  * @param obj_id object id
  * @param color color of the 3D object
  */
-void Object3D::initializeObject3D(const int nb_boards, const int ref_board_id,
-                                  const int obj_id,
-                                  const std::vector<double> color) {
+Object3D::Object3D(const int nb_boards, const int ref_board_id,
```

Comment:
same here

Reply:
Incorporated this feedback in the latest commit